project/SotaBuild.scala

```diff
@@ -89,6 +91,18 @@ object SotaBuild extends Build {
       testOptions in RandomTests += Tests.Argument(TestFrameworks.ScalaTest, "-n", "RandomTest"),
       parallelExecution in Test := true,
       dockerExposedPorts := Seq(8081),
+      dockerCommands := dockerCommands.value.filterNot {
```

Comment:
Something similar to this should work:

```
    .settings(mappings in Docker += (file("deploy/core-entrypoint.sh") -> "entrypoint.sh"))
    .settings(dockerEntrypoint := Seq("/opt/docker/entrypoint.sh"))
    .settings(dockerCommands ++= Seq(
      ExecCmd("ADD", "entrypoint.sh", s"/opt/docker/entrypoint.sh")))
```

Changing the base image might be ok, but they we'd need to create a Dockerfile to create a separate image, because `advancedtelematic/java:openjdk-8-jre` is used for other projects and it's purpose is just to have java installed...